test-data/unit/semanal-errors.test

```diff
@@ -877,7 +877,7 @@ A[TypeVar] # E: Invalid type "typing.TypeVar"
 from typing import TypeVar, Generic
 t = TypeVar('t')
 class A(Generic[t]): pass
-A[1] # E: Type expected within [...]
+A[1] # E: Invalid type. Try using Literal[1] instead?
```

Comment:
For consistency, it's better not to use periods in error messages. Maybe split into an error and a note, which we use for various other suggestions? Here's one way to do it:

```
x.py:4: error: Invalid type "1"
x.py:4: note: Suggestion: use Literal[1] instead of 1?
```

Reply:
I decided to go with messages like:

    x.py:4: error: Invalid type: try using Literal[1] instead?

This seems to match the format of several of the other error messages generated by `typeanal.py` (e.g. "Invalid type: ClassVar nested inside other type").